crates/subspace-core-primitives/src/lib.rs

```diff
@@ -112,6 +113,12 @@ impl fmt::Debug for Blake3HashHex {
     }
 }
 
+impl Borrow<Blake3Hash> for Blake3HashHex {
```

Comment:
This is the first implementation of `Borrow` in this repo, curious why is it necessary? Maybe `Deref` is better?

Reply:
It was used for a `HashMap` by hash, but that's not in the implementation any more, so we don't need `Borrow`. `Deref` and `DerefMut` are already automatically derived.